ddtrace/tracer/option.go

```diff
@@ -311,6 +317,7 @@ func newConfig(opts ...StartOption) *config {
 	c.profilerEndpoints = internal.BoolEnv(traceprof.EndpointEnvVar, true)
 	c.profilerHotspots = internal.BoolEnv(traceprof.CodeHotspotsEnvVar, true)
 	c.enableHostnameDetection = internal.BoolEnv("DD_CLIENT_HOSTNAME_ENABLED", true)
+	c.debugOpenSpans = internal.BoolEnv("DD_TRACE_DEBUG_OPEN_SPANS", false)
```

Comment:
What about the name `DD_TRACE_DEBUG_ABANDONED_SPANS`?
In my mind, the use case for this feature is for customers and support to be able to debug situations where spans were started and then abandoned, and never closed. Not so much just *all* open spans. Plus, this doesn't actually report on all open spans, just old ones.

Reply:
I would also suggest putting all of the configuration for the feature in the same area of the code. So I'd move this enable/disable to right above line 456

Reply:
Makes sense! I'll also change the variable names (i.e. `t.openSpans`). Looking at the code again, it looks like there is an `openSpans` variable on the mocktracer that represents something different, so that might cause some confusion!

Reply:
> I would also suggest putting all of the configuration for the feature in the same area of the code. So I'd move this enable/disable to right above line 456

There is some lines of code (357-359) that takes care of the `...startOptions` input parameter. Having the configuration for `DD_TRACE_DEBUG_ABANDONED_SPANS` ends up overwriting the feature as disabled if I move the line down. Instead, I'll move the if statement that's currently at line 456 up, so it's formatted similar to the partial flushing code!